src/pages/app-development/starter-kit/webhooks-example.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Webhooks example
+description: Learn how to use webhooks with Adobe Commerce Extensibility Starter Kit to connect to a third-party source.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Webhooks example
+
+This reference implementation demonstrates how to expose an entry point in Adobe Commerce to perform a realtime check of the available stock for items in a third-party backoffice system.
+
+<InlineAlert variant="info" slots="text"/>
+
+If you do not want to implement webhooks, comment out the `webhook` section of the Starter Kit's `app.config.yaml` file before running `aio app deploy`.
+
+The runtime action included is in the Starter Kit package in the following location:
+
+- `actions/webhook/check-stock/index.js`
+
+<InlineAlert variant="info" slots="text"/>
+
+You must implement the stock-checking logic on an external service.
+
+## Configure webhook
+
+This runtime action uses the [Adobe Commerce webhook module](../../webhooks/) to allow intercepting flows in Adobe Commerce.
+
+Follow [configure hooks](https://developer.adobe.com/commerce/extensibility/webhooks/hooks/) to modify your `webhooks.xml` file and define the connection between Adobe Commerce and your backoffice system using the `observer.checkout_cart_product_add_before` method.
```

Comment:
Someone in Starter Kit won't have a reference webhooks file. They'll have to manually configure the webhook in the Admin. Change the link target to Admin configuration, and I'll need to update the top of that article to be more inclusive for brand new developers.